TooltipDecorator/TooltipDecorator.js

```diff
@@ -39,7 +39,7 @@ const defaultConfig = {
 	 * resolution.
 	 *
 	 * @type {Number}
-	 * @default 24
+	 * @default 12
```

Comment:
?
```suggestion
	 * @default 24
```

Reply:
I thought the `24` value is incorrect, but after reading the docs more carefully it seems I was wrong, sorry. I changed it back to `12`.